sensor_msgs/Image gate_image
---
sensor_msgs/Image result_image
geometry_msgs/Point frame_center
geometry_msgs/Point gate_center
